x/checkpointing/vote_ext.go

```diff
@@ -116,57 +116,61 @@ func (h *VoteExtensionHandler) VerifyVoteExtension() sdk.VerifyVoteExtensionHand
 			return resAccept, nil
 		}
 
+		extensionSigner := sdk.ValAddress(req.ValidatorAddress).String()
```

Comment:
why not continue as an error if it fails during `VerifyVoteExtension`?

Reply:
My understanding is that Error log means something wrong in your node. But in this case, it just means that some votes are invalid, not something wrong in your node. Imagine that there are bad validators continuously sending invalid votes, people will see a large amount of error logs, which is scary